CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,147 @@
+
+We will respect confidentiality requests all the time for the purpose of protecting individuals who have been harmed.
+
+### Terminology
+
+- Reporter: Person reporting an incident.
+- Reportee: Person being reported.
+
+## Report Handling Process
+
+After receiving a report, the community leaders (or person handling the report) will review the incident report and follow this process to determine the cause and consequences that violated this Code of Conduct and take further actions:
+
+Community leaders will reach out to the reporter to ask for further information if the submitted report is not complete or needs some additional context.
+Community leaders will also ensure that person impacted or harmed by the reported incident is safe and identify how they can be further supported.
+
+A meeting with the reportee will be scheduled where the person handling the report will discuss four questions:
+
+- What happened?
+- What were you thinking at the time?
+- Who has been affected by what you have done? In what way?
+- What do you think you need to do to make things right?
+
+When the incident in question inflicted harm on another person, additional
+questions will be asked on behalf of the person(s) harmed:
+
+- What did you think when you realized what had happened?
+- What impact has this incident had on you and others?
+- What do you think needs to happen to make things right?
+
+The person handling the report may follow up with reportee or reporter after this discussion
+with any additional questions to help them reach a resolution.
+
+### Resolutions
+
+Here are examples of possible resolutions to a report.
+This list is not comprehensive, and any action necessary to reach a fair resolution will be taken as needed.
+Possible resolutions to an incident include:
+
+- a private communication from community leaders, providing
+  clarity around the nature of the violation and an explanation of why the
+  behaviour was inappropriate.
+- an apology to the person harmed or the team as a whole
+- an action plan for the reportee to improve their awareness and correct future behaviours
+- if confirmed by the person reporting, reportee will have no interaction with the people involved, including unsolicited interaction with
+  those enforcing the Code of Conduct, for a specified period. This includes avoiding interactions in community spaces as well as external channels like social media.
+- not publishing the content that violated the Code of Conduct, until necessary changes have been made and agreed upon by the person handling the report.
+- an imposed suspension, such as asking reportee to “take a week off” from the community.
+- a stronger warning with consequences for continued behaviour.
+
+The person handling the report will maintain records of all reports so that they may be reviewed if stronger measures are required when repeated violations occur by the same individual, as well as to improve our Code of Conduct for future participation.
```

Comment:
It sounds like this CoC is designed for people working in a larger institution with more resources. Would we rely on the Turing to provide additional help?